### PR TITLE
Simplify the search bar

### PR DESCRIPTION
**Pull request by dsmilkov**
DEMO: https://huggingface.co/spaces/lilacai/daniel_staging

Before
Screenshot 2023-08-26 at 11 48 46 AM

After:

https://github.com/lilacai/lilac/assets/2294279/64636895-3638-4ea1-a0ce-64cdba69529d

### CODE REVIEW

File: web/blueprint/src/lib/view_utils.ts [hunk 1]
```diff
@@ -276,28 +266,29 @@ export function getSearches(store: DatasetViewState, path?: Path | null): Search
   return (store.query.searches || []).filter(s => pathIsEqual(s.path, path));
 }
 
-function getDefaultSearchPath(datasetStore: DatasetState): Path | null {
+export function getDefaultSearchPath(
```
Comment (nsthorat) [inline web/blueprint/src/lib/view_utils.ts:269]: later we should make this part of the settings

**Review by nsthorat — APPROVED** [review]
Amazing stuff, much nicer.

High level comments:
- Can you round the left of the dropdown and right of the searchbox (rounded-l and rounded-r tailwind classes) to match the rest of our design?
- Can we use a different icon when the index has been computed, or something to separate the overloaded chip icon (which means "it has been computed" in the left dropdown and "it hasn't been computed, compute" in the right dropdown?) Ideally it's like a chip+check but not sure if that icon exists. A color might do it
- I get a funky R/L scroll:
image
- I can check the same filter twice (adding the same filter twice). And then if I remove it by clicking the X on the tag, it removes both:
image
- Clicking on near_dip.cluster_id does nothing (same with num_characters, readability, TTR, frac_non_ascii):
image
- When I add a filter for has emails for two columns, we dont show the column in the pill, we may want to group the pills by the column so we dont show the path for every pill:
image

File: web/blueprint/src/lib/view_utils.ts (hunk 1 above)
Comment (dsmilkov) [reply web/blueprint/src/lib/view_utils.ts:269]: ack

**Comment by dsmilkov** [comment]
great comments -- addressed all but 2

> Clicking on near_dip.cluster_id does nothing (same with num_characters, readability, TTR, frac_non_ascii):

it applies a sort DESC, might not be obvious, open to ideas on how to clarify that

> When I add a filter for has emails for two columns, we dont show the column in the pill

**Pull request merged**